serverlessworkflow/modules/ROOT/pages/cloud/index.adoc

```diff
@@ -0,0 +1,68 @@
+= {product_name} applications in Kubernetes
+:compat-mode!:
+// Metadata:
+:description: Workflow applications in Kubernetes
+:keywords: cloud, kubernetes, docker, image, podman, openshift, pipelines
+// other
+
+{product_name} is evolving as platform to build and deploy workflow applications in Kubernetes clusters.
+
+You have two options to deploy your workflow applications. Either via the traditional Java Quarkus project, or using the newly {operator_name}.
+
+The cards below list every feature included in the platform to deploy workflow applications either with Quarkus or the Operator.
+
+[NOTE]
+====
+Eventually these two options will converge to the {operator_name}. So if you opt in to use Quarkus now and manually deploy your workflows, bear in mind that it's on the project's roadmap to integrate the Quarkus experience with the Operator.
```

Comment:
```suggestion
Eventually these two options will converge, the {operator_name} will also be able to handle full Quarkus projects. So if you opt in to use Quarkus now and manually deploy your workflows, bear in mind that it's on the project's roadmap to integrate the Quarkus experience with the Operator.
```

Reply:
Full quarkus projects, maybe just Quarkus is better?